packages/cardpay-reward-programs/cardpay_reward_programs/rules/dummy_rule.py

```diff
@@ -0,0 +1,113 @@
+import pandas as pd
+import duckdb
+from cardpay_reward_programs.rule import Rule
+
+class DummyRule(Rule):
+    """
+    This rules rewards acounts with 
+    """
+
+    def __init__(self, core_parameters, user_defined_parameters):
+        super(DummyRule, self).__init__(core_parameters, user_defined_parameters)
+
+    def set_user_defined_parameters(
+        self,
+        token,
+        base_reward,
+        transaction_factor, #not used, just for compatibility with parameters.json
+        spend_factor, #not used, just for compatibility with parameters.json
```

Comment:
We discussed this in a call briefly, but these parameters can be removed - the json files are config per rule and don't need to work across all of them

Reply:
Solved through commit 9e62fc731